base-theme/assets/css/variables.scss

```diff
@@ -24,6 +24,7 @@ $card-background: #fff;
 $font-gray: #888;
 $podcast-border-gray: #9c9d9e;
 $medium-gray: #c2c0bf;
+$label-gray: #3d3d3d;
```

Comment:
ayy what a mess of colors we have here - this file defines no less than 22 different gray colors. What do you think of using this opportunity to get it down to a more reasonable level? In my mind there should probably only be 3 or so: a font gray, a background / unfocused font gray, and a border color.

maybe at least while we're in here we could delete the ones we're not using at all right now? it looks like `$bg-gray` isn't used, possibly others

Reply:
@abdulkdawson Any objections to me merging some grays together until we get down to 3?

Reply:
cc @abdulkdawson. Do we have a palette that we are using?

Reply:
@noisecapella I have no objections at all.

Reply:
@noisecapella These are only shades of grey that I've been using. #3D3D3D, #F5F5F5 and #9C9D9E